src/components/CommentCard.jsx

```diff
@@ -0,0 +1,18 @@
+
+function CommentCard ( { comment } ) {
+
+    const unformattedDate = new Date(comment.created_at);
+    const options = {
+        year: "numeric",
+        month: "long",
+        day: "numeric"
+    }
+    const formattedDate = unformattedDate.toLocaleString('en-GB', options);
```

Comment:
Nice! I don't think we've shown it on the course, so props to you for going off and reading some Documentation 😄